src/components/Synthetics/MarketCard/MarketCard.tsx

```diff
@@ -96,9 +97,22 @@ export function MarketCard({ marketInfo, allowedSlippage, isLong, isIncrease }:
         <br />
         <br />
         {oppositeFeeElement}
+        {isMarketWithAdaptiveFundingRate && (
+          <>
+            <br />
+            <br />
+            <Trans>
+              This market uses an Adaptive Funding Rate. The Funding Rate will adjust over time depending on the ratio
+              of longs and shorts.
+              <br />
+              <br />
+              <ExternalLink href="https://docs.gmx.io/docs/trading/v2/#adaptive-funding">Read more</ExternalLink>.
+            </Trans>
```

Comment:
Will "Read more" be translated by `<Trans>` that way?
feels like it should have separate `<Trans>` in children

Reply:
It will be translated, Trans takes care of all the text element inside.

The message will look like this `This market uses an Adaptive Funding Rate. The Funding Rate will adjust over time depending on the ratio of longs and shorts.<0/><1/><2>Read more</2>.`